Sources/Reports.php

```diff
@@ -319,38 +321,24 @@ function BoardPermissionsReport()
 
 	// Fetch all the board names.
 	$request = $smcFunc['db_query']('', '
-		SELECT id_board, name, id_profile
-		FROM {db_prefix}boards
+		SELECT id_board, b.name, id_profile
```

Comment:
since using join, would be good to use on every column the table alias,
otherwise it's not clear from where the column is coming